.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: test
+
+on:
+  workflow_dispatch:
```

Comment:
Do we need a workflow_dispatch here? I mean it doesn't hurt to have it. Just asking if we want a clean setup or just keep it

Reply:
Yeah you are right, we can remove this.

Reply:
```suggestion
```

Reply:
I leave the honour of removing it to you :trollface: